app/frontend/src/styles/application/form.scss

```diff
@@ -19,3 +19,30 @@
   color: $govuk-link-hover-colour;
   text-decoration: underline;
 }
+
+.start-date {
+  display: flex;
+  @media only screen and (max-width: 882px) {
+    flex-wrap: wrap;
+
+    .govuk-form-group {
+      flex-basis: 100%;
+    }
+  }
+
+  .govuk-checkboxes__item {
+    margin-bottom: govuk-spacing(6);
+    margin-left: govuk-spacing(4);
+  }
+
+  .checkbox-starts-asap {
```

Comment:
this would become `clear-form__checkbox`

Reply:
BEM = `block__element--modifier`

Reply:
whilst we dont really harness the full benefit of BEM in TV app, some kind of naming convention is better than none. I do have plans to improve the naming of everything at some point soon and trying to use BEM now will aid that task